bin/addRedirectURItoConnect.ts

```diff
@@ -0,0 +1,54 @@
+import { updateApplication } from "../lib/commands/updateApplication";
+import { getApplication } from "../lib/queries/getApplication";
+import { config } from "../src/config";
+
+const addingRedirectURIToConnect = async (): Promise<any> => {
```

Comment:
```suggestion
const addingRedirectURIToConnect = async (): Promise<void> => {
```
Same on the other one.